packages/core/base/package.json

```diff
@@ -35,6 +35,9 @@
         "@solana/web3.js": "^1.58.0"
     },
     "dependencies": {
+        "@solana/wallet-standard-features": "^1.0.0",
+        "@wallet-standard/base": "^1.0.1",
+        "@wallet-standard/features": "^1.0.1",
```

Comment:
This is the part of this PR that's ugly, IMO. Wallet Adapter's `base` package has avoided adding any new dependencies since it was created. These _are_ just types, but I still don't love it.

Reply:
I wish there was some indication that the package is "types-only"

Reply:
Yeah. I thought of putting them in `devDependencies`, but I'm not 100% sure what downstream effects that might have.

Reply:
Hmm. Since types are dev-only, even in packages that depend on this one, can you add it to devDependencies and still have everything work?

Reply:
if you put it in dev deps they won't be installed when yoyr package is installed, so the person who uses your package will have issues with types

Reply:
Oh, yeah. Of course. Dang.